Pod/ILGClasses/ILGClasses.h

```diff
@@ -8,6 +8,8 @@
 
 #import <Foundation/Foundation.h>
 
+NS_ASSUME_NONNULL_BEGIN
```

Comment:
At least when I was doing it here, I tried to do an `if let` unwrap on `NSSet` and it told me that I couldn't do that with something that was already force-unwrapped.